### ISSUE TITLE
Merge the two file upload field type in to one.

### ISSUE THREAD

**Issue by frjo** [OpenTechFund/hypha#2102]
There are now two stream fields for uploading files, "File field" and "Multiple file field".

We should remove/hide "File field" from being added to new forms. Instead add a setting on "Multiple file field" to set how many fields are allowed. 1-10 or unlimited are my suggestion for options.

Or is it never needed to limit the number of uploads? If that is the case the new setting is not needed.

Lets use the icon now used for "File field" on "Multiple file field" as well when this is worked on.

**Comment by ei8fdb** [comment]
> Instead add a setting on "Multiple file field" to set how many fields are allowed. 1-10 or unlimited are my suggestion for options.

My input is the "set how many fields are allowed" is low priority as I think it would be difficult to predict this usage.

> Or is it never needed to limit the number of uploads? If that is the case the new setting is not needed.

I agree - for now the new setting of "how many fields are allowed" is not needed. 

This could be handled with some simple content:
- "please upload [the description of the single file you want to see], e.g. "please upload your CV"
- "please upload [the description of the files you want to see], e.g. "please upload the CVs of your team members"

**Comment by fourthletter** [comment]
Could we close because it falls under an overarching issue of  #2275

**Comment by ei8fdb** [comment]
@fourthletter how is it connected to that? (Not sure I follow!)

**Comment by fourthletter** [comment]
You're right! It's unrelated. I thought #2275 was about having preset templates for creating new application forms in WagTail. I assumed the correct document/file upload would be incorporated into this template.

**Comment by fourthletter** [comment]
> Or is it never needed to limit the number of uploads? If that is the case the new setting is not needed.

I recently started creating new applications for Funds, and see a value in limiting how many documents an applicant could upload at any one time. However, I'm willing to leave this up to standard best practice.

A secondary issue are the icons for "File field" on "Multiple file field" are unclear to users.
![image](https://user-images.githubusercontent.com/20019656/163976491-22b84859-edb1-4c4e-9cec-19bac83f7b4e.png)

**Comment by ei8fdb** [comment]
Are there any objections to the following:

- removing the single file upload, 
- keeping the multiple file upload and then
- deal with the maximum file upload limit in content

This would be the best experience for users and simplest to implement (I think!).

If this doesn't work, then we can implement some max. file limit control.

Objections?

**Comment by fourthletter** [comment]
@sandeepsajan0 this may seem like a minor issue, but this upload field has plagued many users for centuries. 

- [ ] removing the single file upload,
- [ ] keeping the multiple file upload and then
- [ ] deal with the maximum file upload limit in content

Example: This could be handled with some simple content:
"please upload [the description of the upload field you want to see], e.g. "please upload your CV"
"please upload [the description of the upload field you want to see], e.g. "please upload the CVs of your team members"

**Comment by fourthletter** [comment]
Standup April 21 - thank you @sandeepsajan0 for your question

During standup we decided the best course of action moving forward is to disable the "File field" as an option when users are creating forms in WagTail. Providing only the "Multiple file field" should be sufficient. 

We cannot remove the single file block code because existing forms associated with Funds/Labs are using this field. Disabling this field so it does not show up as a field will prevent this field from being added to forms in the future.

**Comment by fourthletter** [comment]
Standup April 26

During we discussed the technical debt for continuing to work on this issue, and the potential benefit of removing the single finle upload button.

**Comment by fourthletter** [comment]
Looks great! Thank you @sandeepsajan0 💯 

![image](https://user-images.githubusercontent.com/20019656/165471139-f8f7c52d-130a-4668-b1dd-87e010cce518.png)
